docs/tools-reference/custom-kernels-distros/run-a-distributionsupplied-kernel-with-pvgrub.md

```diff
@@ -564,46 +564,58 @@ CentOS 5
 
         Linux li181-194 2.6.18-194.26.1.el5xen #1 SMP Tue Nov 9 14:13:46 EST 2010 i686 i686 i386 GNU/Linux
 
-Fedora 17
+Fedora 21
 ---------
 
-1.  Make sure your package repositories and installed packages are up to date by issuing the following command:
+1. Make sure your package repositories and installed packages are up to date by issuing the following command:
```

Comment:
The extra space after the number is required for our site to properly see the command below without any extra spaces prepended to it.